src/libproc_macro/lib.rs

```diff
@@ -66,6 +66,16 @@ pub struct LexError {
     _inner: (),
 }
 
+#[stable(feature = "proc_macro_lexerror_impls", since = "1.43.0")]
```

Comment:
I think it should be next version now.

```suggestion
#[stable(feature = "proc_macro_lexerror_impls", since = "1.44.0")]
```

Reply:
r=me with this applied.